airbyte-cdk/bulk/core/load/src/test/kotlin/io/airbyte/cdk/load/task/DestinationTaskLauncherTest.kt

```diff
@@ -544,6 +538,42 @@ class DestinationTaskLauncherTest<T> where T : LeveledTask, T : ScopedTask {
             taskLauncher.run()
             Assertions.assertTrue(mockScopeProvider.didKill)
         }
-        mockExceptionHandler.callbacks.receive().invoke()
+        taskLauncher.handleTeardownComplete(success = false)
+    }
+
+    @Test
```

Comment:
I found it impossible to get either of these tests to work with mockk. I kept getting bizarre unrelated errors (like failure to mock stuff inside of some method calls regardless of whether I stubbed them). Nor could I find any way to force a failure inside a task in a context that would trigger the callback.